resources/resources/pe/blockdefinition.json

```diff
@@ -4983,4 +3165,20 @@
 {"data":4, "id":466, "name":"minecraft:jigsaw"},
 {"data":5, "id":466, "name":"minecraft:jigsaw"},
 {"data":6, "id":466, "name":"minecraft:jigsaw"},
-{"data":7, "id":466, "name":"minecraft:jigsaw"}]
\ No newline at end of file
+{"data":7, "id":466, "name":"minecraft:jigsaw"},
+{"data":0, "id":467, "name":"minecraft:lit_blast_furnace"},
+{"data":1, "id":467, "name":"minecraft:lit_blast_furnace"},
+{"data":2, "id":467, "name":"minecraft:lit_blast_furnace"},
+{"data":3, "id":467, "name":"minecraft:lit_blast_furnace"},
+{"data":4, "id":467, "name":"minecraft:lit_blast_furnace"},
+{"data":5, "id":467, "name":"minecraft:lit_blast_furnace"},
+{"data":6, "id":467, "name":"minecraft:lit_blast_furnace"},
+{"data":7, "id":467, "name":"minecraft:lit_blast_furnace"},
+{"data":0, "id":468, "name":"minecraft:lit_smoker"},
+{"data":1, "id":468, "name":"minecraft:lit_smoker"},
+{"data":2, "id":468, "name":"minecraft:lit_smoker"},
+{"data":3, "id":468, "name":"minecraft:lit_smoker"},
+{"data":4, "id":468, "name":"minecraft:lit_smoker"},
+{"data":5, "id":468, "name":"minecraft:lit_smoker"},
```

Comment:
There's a lot of changes in block definitions. Are you sure they are correct? Or, rather, do you trust them?

Reply:
👍